pkg/controllers/resources/services/syncer.go

```diff
@@ -24,8 +24,7 @@ func New(ctx *synccontext.RegisterContext) (syncer.Object, error) {
 	return &serviceSyncer{
 		NamespacedTranslator: translator.NewNamespacedTranslator(ctx, "service", &corev1.Service{}),
 
-		syncServiceSelector: ctx.Options.SyncServiceSelector,
-		serviceName:         ctx.Options.ServiceName,
+		serviceName: ctx.Options.ServiceName,
 	}, nil
```

Comment:
You removed the assignment of the `syncServiceSelector` property here^
but it is still part of the `seviceSyncer` struct - https://github.com/loft-sh/vcluster/blob/cf1d01c2ddf042e8be66a9f82d686976caecf7c7/pkg/controllers/resources/services/syncer.go#L34
and more importantly, it is still checked during service translation, which prevents sync of the selector - https://github.com/loft-sh/vcluster/blob/cf1d01c2ddf042e8be66a9f82d686976caecf7c7/pkg/controllers/resources/services/translate.go#L12-L13
This definitely needs to be fixed, and it is what causes CoreDNS e2e tests to fail.